### PR TITLE
cloud: add integration with GPT

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/data-service-manage-data-app.md [hunk 1]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
...
+
+To integrate your Data App with GPT, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, locate your target Data App and click the name of your target Data App to view its details.
+3. In the **Integration with GPT** area, click **Get Configuration**.
+4. In the displayed dialog box, you can see the following fields:
+
+    - **API Specification URL**: copy the URL of the OpenAPI Specification of your Data App. For more information, see [Use the OpenAPI Specification](#use-the-openapi-specification).
+    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. Click **Encode** to encode the API Key. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```
Comment (jusong) [inline tidb-cloud/data-service-manage-data-app.md:76]: [Click **Encode**] 这个按钮去掉了，输入apikey之后在下面会自动生成编码之后的内容

Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:76]:
```suggestion
    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```

Comment (hfxsd) [reply tidb-cloud/data-service-manage-data-app.md:76]: Create API Key 按钮也删掉了是吗？

Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:76]:
```suggestion
    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```

File: tidb-cloud/data-service-manage-data-app.md [hunk 2]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
...
 
+### Integrate your Data App with GPT
+
+You can integrate your Data App with GPT to enhance your applications with intelligent capabilities.
+
+To integrate your Data App with GPT, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, locate your target Data App and click the name of your target Data App to view its details.
+3. In the **Integration with GPT** area, click **Get Configuration**.
```
Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:72]:
```suggestion
3. In the **Integration with GPTs** area, click **Get Configuration**.
```

File: tidb-cloud/data-service-manage-data-app.md [hunk 3]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
 
 For more information, see [Deploy automatically with GitHub](/tidb-cloud/data-service-manage-github-connection.md).
 
+### Integrate your Data App with GPT
```
Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:64]:
```suggestion
### Integrate your Data App with GPTs
```

File: tidb-cloud/data-service-manage-data-app.md [hunk 4]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
...
+
+You can integrate your Data App with GPT to enhance your applications with intelligent capabilities.
+
+To integrate your Data App with GPT, perform the following steps:
```
Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:68]:
```suggestion
To integrate your Data App with GPTs, perform the following steps:
```

File: tidb-cloud/data-service-manage-data-app.md [hunk 5]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
 
 For more information, see [Deploy automatically with GitHub](/tidb-cloud/data-service-manage-github-connection.md).
 
+### Integrate your Data App with GPT
+
+You can integrate your Data App with GPT to enhance your applications with intelligent capabilities.
```
Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:66]:
```suggestion
You can integrate your Data App with GPTs to enhance your applications with intelligent capabilities.
```

**Review by jusong — APPROVED** [review]

File: tidb-cloud/data-service-manage-data-app.md [hunk 6]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
 
 For more information, see [Deploy automatically with GitHub](/tidb-cloud/data-service-manage-github-connection.md).
 
+### Integrate your Data App with GPTs
+
+You can integrate your Data App with GPTs to enhance your applications with intelligent capabilities.
```
Comment (qiancai) [inline tidb-cloud/data-service-manage-data-app.md:66]:
```suggestion
You can integrate your Data App with [GPTs](https://openai.com/blog/introducing-gpts) to enhance your applications with intelligent capabilities.
```

File: tidb-cloud/data-service-manage-data-app.md [hunk 7]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
...
+To integrate your Data App with GPTs, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, locate your target Data App and click the name of your target Data App to view its details.
+3. In the **Integration with GPTs** area, click **Get Configuration**.
+4. In the displayed dialog box, you can see the following fields:
+
+    - **API Specification URL**: copy the URL of the OpenAPI Specification of your Data App. For more information, see [Use the OpenAPI Specification](#use-the-openapi-specification).
+    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
+    - **API Key Encoded**: copy the base64 encoded string equivalent of the API Key you have provided.
```
Comment (qiancai) [inline tidb-cloud/data-service-manage-data-app.md:77]:
```suggestion
    - **API Key Encoded**: copy the base64 encoded string equivalent to the API key you have provided.

```

File: tidb-cloud/data-service-manage-data-app.md [hunk 8]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
...
+
+To integrate your Data App with GPTs, perform the following steps:
+
+1. Navigate to the [**Data Service**](https://tidbcloud.com/console/data-service) page of your project.
+2. In the left pane, locate your target Data App and click the name of your target Data App to view its details.
+3. In the **Integration with GPTs** area, click **Get Configuration**.
+4. In the displayed dialog box, you can see the following fields:
+
+    - **API Specification URL**: copy the URL of the OpenAPI Specification of your Data App. For more information, see [Use the OpenAPI Specification](#use-the-openapi-specification).
+    - **API Key**: provide the API key of your Data App. If you do not have an API Key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```
Comment (qiancai) [inline tidb-cloud/data-service-manage-data-app.md:76]:
```suggestion
    - **API Key**: enter the API key of your Data App. If you do not have an API key yet, click **Create API Key** to create one. For more information, see [Create an API key](/tidb-cloud/data-service-api-key.md#create-an-api-key).
```

File: tidb-cloud/data-service-manage-data-app.md [hunk 9]
```diff
@@ -61,6 +61,22 @@ You can edit the name, version, and description of a Data App. To edit Data App
...
+    - **API Key Encoded**: copy the base64 encoded string equivalent of the API Key you have provided.
+5. Copy the API Specification URL and the encoded API Key, and use them in the GPT configuration.
```
Comment (qiancai) [inline tidb-cloud/data-service-manage-data-app.md:78]: 第 4 步里已经告诉要 copy 了，这里要再次 copy 吗

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-cloud/data-service-manage-data-app.md (hunk 9 above)
Comment (hfxsd) [inline tidb-cloud/data-service-manage-data-app.md:78]:
```suggestion
5. Now you can use the coped API Specification URL and the encoded API key in the GPT configuration.
```

Comment (hfxsd) [reply tidb-cloud/data-service-manage-data-app.md:78]: 改成下面这样了。PTAL

**Review by heibaijian — APPROVED** [review]

**Pull request merged**